jobs/config.json

```diff
@@ -2890,6 +2909,25 @@
       "sig-instrumentation"
     ]
   },
+  "ci-kubernetes-e2e-gce-kubedns-performance": {
+    "args": [
+      "--cluster=gce-kubedns-performance",
+      "--env=CLUSTER_DNS_CORE_DNS=false",
```

Comment:
If this is default, we might just eliminate this line.

Reply:
It is the default now - but next release the default may be true, in which case we wont need to update this env setting if we explicitly set it to false.

Reply:
Sgtm